src/api/channels.js

```diff
@@ -132,24 +132,18 @@ export function isMaxBodyDaysValid(channel) {
  * Creates a new channel
  */
 export async function addChannel(ctx) {
-  // Test if the user is authorised
-  if (authorisation.inGroup('admin', ctx.authenticated) === false) {
-    utils.logAndSetResponse(
-      ctx,
-      403,
-      `User ${ctx.authenticated.email} is not an admin, API access to addChannel denied.`,
-      'info'
-    )
-    return
-  }
+  try {
+    // Test if the user is authorised
+    const authorised = await utils.checkUserPermission(ctx, 'addChannel', 'channel-manage-all')
```

Comment:
The add action should also be controlled by `channel-manage-specified`

Reply:
We cant control this by `channel-manage-specified` because the channel 's `Id` is created when a channel is created on mongo

Reply:
Ah, I see. Of course.